src/command_modules/azure-cli-storage/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.2.5
++++++
+* Improvements and fixes for storage copy commands.
```

Comment:
This should be more specific, at least regarding what parameters were added or behaviors changed.

Reply:
-No parameters added, a lot of missed corner cases taken care of however. I didn't want to bloat the history with statements like, if input1, input2 were specified and input3 was not, do a certain action.
-Let me know if there's anything in the description of this PR that you'd like to see in the history.

Reply:
Agreed, it is hard to put in precise while compact message here, but maybe we can mention a few words for the auth simplification if we can. I would leave the English to you guys to figure out :)

Reply:
If it is closing usability issues that customers have reported, it would be good to call that out--so they can see "oh that thing that wasn't working is now fixed"

Reply:
If this is solving multiple problems, it won't have to hurt multiple (2 or 3) history.rst entries right?